container_cloud_config/templates/dockersystemd.yaml

```diff
@@ -43,8 +47,8 @@
     {% else -%}
     Restart={{ restart_policy }}
     {%- endif %}
-    TimeoutStartSec=600
-    TimeoutStopSec=2000
+    TimeoutStartSec={{ timeout_start_sec|default(600) }}
```

Comment:
These timeouts are way higher than the defaults for regular units and infinitely higher than the default for oneshots. Can we make this optional? I.e. only include this bit if they are defined?

Reply:
Oh, I missed that they were already really high, heh.